src/components/TipTapEditor.tsx

```diff
@@ -0,0 +1,70 @@
+'use client'
+import React, { useEffect } from 'react';
+import { EditorContent, useEditor } from '@tiptap/react';
+import { StarterKit } from '@tiptap/starter-kit';
+import TipTapMenuBar from '@/components/TipTapMenuBar';
+import { Button } from './ui/button';
+import Underline from '@tiptap/extension-underline';
+import { useDebounce } from '@/lib/useDebounce';
+import { useMutation } from '@tanstack/react-query';
+import axios from 'axios';
+import { LoaderPinwheel } from 'lucide-react';
+type Props = {
+    note: any;
+}
+/*
+    Description ->  React Component for creating a TipTap Editor.
+*/ 
+const TipTapEditor = ({ note }: Props) => {
+    const [content, setContent] = React.useState(note.content || 'Start typing...');
```

Comment:
Start Typing does not go away when I click to start typing. Can we make it go away when we click on text area ?
Just a suggestion nothing important.

Reply:
It's not a placeholder but I could look into making it into one in the future.